src/search/utils/clause.ts

```diff
@@ -48,6 +21,71 @@ export const createEmptyClause = (): Clause => ({
   queryInput: {},
 });
 
+export const createPreSelectedClauses = () : Clause[] => [
+  {
+    id: v1(),
+    logicOperator: Operator.AND,
+    searchTerm: {
```

Comment:
I'm aware that we're creating some work for ourselves if the `searchTerms` changes. As I think we're testing this pre-filled advanced search with users it's fine to leave as is but I'll create a jira that states if we're keeping this it's best to use the entries from the fetched `searchTerms` rather than using the static values used here.

Reply:
Great, that's fine!